src/commands/Utilities/help.js

```diff
@@ -80,7 +80,7 @@ class HelpCommand extends Command {
                     this.client.user.displayAvatarURL()
                 );
 
-            for (const category of this.handler.categories.values()) {
+            for (const category of this.handler.categories.filter((categ) => categ.id != 'Bot Owner Only').values()) {//filter will hide owner commands from all users
```

Comment:
We don't wanna hide commands of a developer if the commands are ran by a developer right ?

Reply:
so an if statement maybe? like a function for filter command?